### ISSUE TITLE
Learn Proto.IO

### ISSUE THREAD

**Issue by jekram** [Cloudkibo/Intern#20]
@AsadUllahBinTahir 

Doing a great UI design is a very critical element. So as part of your training, I would like you to learn Proto.io.

[Mock-up from HotLine.pptx](https://github.com/Cloudkibo/Intern/files/185660/Mock-up.from.HotLine.pptx)

You can signup for a free trial account at www.proto.io. 

Next steps:

1. They have lots of training video that you should watch.
2. Develop sample apps from tutorials. 
3. Next, you should develop as a prototype the attached 

Deliverables:

1. Sample from Tutotial
2. Mockup from step three above

Let's target to finish this 31st of March.

Regards

Jawaid

**Comment by AsadUllahBinTahir** [comment]
Today I watched some proto.io tutorials and studied some documents about the mobile app design theory.

**Comment by jekram** [comment]
Thanks

**Comment by AsadUllahBinTahir** [comment]
I worked on apps prototypes given in prototype apps tab. I also read some articles about the design theory.

**Comment by jekram** [comment]
Thanks for the update

**Comment by AsadUllahBinTahir** [comment]
I have done some of the app prototypes. Although I have not completed all of them, but I have understood the functionality and the implementation of features in proto.io.
Today I am going to start ReactJS.

**Comment by jekram** [comment]
Thanks for the update. What will the ETA for this

"Deliverables:

Sample from Tutorials
Mockup from step three above
Let's target to finish this 31st of March."

**Comment by jekram** [comment]
@AsadUllahBinTahir 

I have not seen any update here. What is the status

**Comment by AsadUllahBinTahir** [comment]
@jekram :
I have completed all the tutorials of Proto.io. Following are the sample prototypes I have made after watching the tutorials. I have not developed all prototypes given in the tutorials because some assets are supposed to be built on photoshop for use in Prototype.
1- Pandora app 
        https://asadullah1.proto.io/share/?id=055b2754-d546-4124-8fc4-211cb213d0a8&v=1
2- Tumblr composed animation
        https://asadullah1.proto.io/share/?id=438890a5-458f-4326-aeaa-e1004fc5ffa5&v=1
3- Facebook Page slide
        https://asadullah1.proto.io/share/?id=cde28705-870f-4d4d-a216-008ad6433d31&v=1
4- Hotline Mockup
        https://asadullah1.proto.io/share/?id=1810c345-04ac-451a-a521-2e685a623b88&v=1

Due to unavailability of photoshop assets, I have left some app prototypes Though I have understood the functionality of each function in proto io. by watching all tutorial videos. 

I am confident enough of my proto io skill. This issue can be closed now.
